src/Serilog.Sinks.OpenTelemetry/Sinks/OpenTelemetry/ProtocolHelpers/PrimitiveConversions.cs

```diff
@@ -128,24 +128,20 @@ public static AnyValue ToOpenTelemetryMap(StructureValue value)
         
         foreach (var prop in value.Properties)
         {
-            if (seen.Contains(prop.Name))
-                continue;
-
-            seen.Add(prop.Name);
-
-            var v = ToOpenTelemetryAnyValue(prop.Value);
-            var kv = new KeyValue
+            if (seen.Add(prop.Name))
             {
-                Key = prop.Name,
-                Value = v
-            };
-            kvList.Values.Add(kv);
+                kvList.Values.Add(new KeyValue
+                {
+                    Key = prop.Name,
+                    Value = ToOpenTelemetryAnyValue(prop.Value)
+                });
+            }
```

Comment:
@bartelink looks like it's here now :point_up:

Reply:
Looks lovely (though I still can't fathom why one would still want egregious braces!)